tests/CMakeLists.txt

```diff
@@ -67,10 +67,11 @@ set(INSTALL_GTEST CACHE BOOL OFF)
 FetchContent_MakeAvailable(googletest)
 
 find_package(OpenMP REQUIRED)
+find_package(GTest CONFIG REQUIRED)
 
 target_link_libraries(faiss_test PRIVATE
   OpenMP::OpenMP_CXX
-  gtest_main
+  GTest::gtest_main
```

Comment:
I think this breaks cmake

```
cmake . -B build   -DFAISS_ENABLE_GPU=OFF   -DBLA_VENDOR=OpenBLAS   -DFAISS_ENABLE_PYTHON=OFF   -DBUILD_TESTING=ON   -DBUILD_SHARED_LIBS=ON -DBLAS_LIBRARIES=/home/ckerchner/install/lib/libopenblas.so -DLAPACK_LIBRARIES=/home/ckerchner/install/lib/libopenblas.so
-- The CXX compiler identification is IBMClang 17.1.2.3
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /opt/IBM/openxlC/17.1.2/bin/ibm-clang++_r - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found OpenMP_CXX: -openmp
-- Found OpenMP: TRUE
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Could NOT find MKL (missing: MKL_LIBRARIES)
-- Found BLAS: /home/ckerchner/install/lib/libopenblas.so
-- Found LAPACK: /home/ckerchner/install/lib/libopenblas.so
CMake Error at tests/CMakeLists.txt:63 (find_package):
  Could not find a package configuration file provided by "GTest" with any of
  the following names:

    GTestConfig.cmake
    gtest-config.cmake

  Add the installation prefix of "GTest" to CMAKE_PREFIX_PATH or set
  "GTest_DIR" to a directory containing one of the above files.  If "GTest"
  provides a separate development package or SDK, be sure it has been
  installed.
```

Reply:
https://stackoverflow.com/questions/54272250/cmake-cant-find-gtest-1-8-in-ubuntu-18-10

Should it be "GTest" or "gtest"?